src/UnitTests/IMappingExpression/NonGenericProjectEnumTest.cs

```diff
@@ -73,7 +73,7 @@ public void ProjectingEnumToString()
         {
             var customers = new[] { new Customer() { FirstName = "Bill", LastName = "White", CustomerType = CustomerType.Vip } }.AsQueryable();
 
-            var projected = _mapper.Map<CustomerDto[]>(customers);
+            var projected = customers.ProjectTo<CustomerDto>(_mapper.ConfigurationProvider);
```

Comment:
IDK I was just saying if the great @jbogard didn't know then I guess my assumption is correct by default.